backend/src/main/java/com/bakdata/conquery/models/datasets/Table.java

```diff
@@ -71,7 +72,7 @@ public class Table extends LabeledNamespaceIdentifiable<TableId> implements Init
 	private Column primaryColumn;
 
 	@JsonView(View.InternalCommunication.class)
-	@Setter(AccessLevel.PRIVATE)
+	@Setter(value = AccessLevel.PUBLIC, onMethod_ = @VisibleForTesting)
```

Comment:
Wild

Reply:
Ja ich weiß, das zu fixen ist aber out of scope, das grundlegende Problem ist hier, dass wir nur einen Deserializer auf Shard-seite haben für alle datasets. Alternative wäre sonst den Test deutlich komplizierter zu machen